packages/verification-vue-component/src/components/Verification/items/cards/VerificationCard.vue

```diff
@@ -158,15 +158,18 @@ export default {
         verificationItem: {
             type: Object,
             required: true
+        },
+        net: {
+            type: String,
+            required: true
         }
     },
     computed: {
-        net() {
-            if (this.$parent.$parent.certifactionEthVerifier.certifactionEthClient.eth.currentProvider.host.indexOf('ropsten') >= 0) {
-                return 'ropsten.etherscan.io'
-            } else {
-                return 'etherscan.io'
-            }
+        registerEvents() {
+            return this.verificationItem.events.filter(event => ['register', 'certify'].includes(event.scope) > 0)
```

Comment:
Instead of includes I would use indexOf() >= 0 as indexOf() is implemented way longer and likely to be more performant and it's less likely to get polyfill code. Because of the "not dead" in the browserlist we still get polyfill-code for IE11 for the next 2 years.

Reply:
Fixed in last commit